tests/dummy/app/locales/en/translations.js

```diff
@@ -296,7 +296,8 @@ Ember.$.extend(true, translations, {
       'name-caption': 'Name',
       'moderated-caption': 'Moderated',
       'parent-caption': 'Parent',
-      'localizedTypes-caption': 'Localized types'
+      'localizedTypes-caption': 'Localized types',
```

Comment:
I suppose we should use single keys style for all keys in i18n translations files: kebab case instead of camel case (localized-types-caption instead of localizedTypes-caption).